src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+
```

Comment:
https://github.com/mate-academy/jv-oop/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code
```suggestion
    public abstract void doWork();

    public abstract void stopWork();
```